content/guides/ai-studio/create-agents.md

```diff
@@ -0,0 +1,82 @@
+---
+rank: 2
+related_guides:
+  - authentication/tokens/developer-tokens/
+  - box-ai/ai-tutorials/ask-questions
+  - box-ai/ai-tutorials/default-agent-overrides
+  - box-ai/ai-tutorials/generate-text
+  - box-ai/ai-tutorials/extract-metadata-structured
+  - box-ai/ai-tutorials/extract-metadata
+---
+# Create agents
+
+The `POST /2.0/post_ai_agents` endpoint allows you to create a new, custom [AI agent][agents]. 
+
+## Send a request
+
+To send a request, use the `POST /2.0/post_ai_agents` endpoint.
+
+Make sure you have generated the developer token
+to authorize your app. See [getting started with Box AI][prereq]
+for details.
+
+<Samples id='post-ai-agents' />
+
+### Parameters
+
+To make a call, you must pass the following parameters. Mandatory parameters are in **bold**.
+
+| Parameter| Description| Example|
+|--------|--------|-------|
+| **`type`** | The type of agent used to handle queries. | `ai_agent` |
+| **`access_state`** | The state of the AI Agent. Value is one of `enabled` `disabled`. | `enabled` |
+| `ask` | The AI Agent to be used for ask. | `ask` |
+| `extract` | The AI Agent to be used for extraction. | |
+| **`is_enterprise_default`** | If the agent is default for enterprise. | `false` |
+| **`name`** | The name of the AI Agent. | `My AI Agent` |
+| `text_gen` | The AI agent used for generating text. | |
+
+## Update AI agent
```

Comment:
I wonder if update and delete should be on separate pages, as the url includes just create-agents scope?